src/functions/module_lookup.ts

```diff
@@ -213,6 +213,43 @@ export const HLOOKUP: AddFunctionDescription = {
   isExported: true,
 };
 
+// -----------------------------------------------------------------------------
+// INDEX
+// -----------------------------------------------------------------------------
+export const INDEX: AddFunctionDescription = {
+  description: _lt("Returns the content of a cell, specified by row and column offset."),
+  args: [
+    arg("reference (range)", _lt("The range of cells from which the values are returned.")),
+    arg(
+      "row (number, optional, default=0)",
+      _lt("The index of the row to be returned from within the reference range of cells.")
+    ),
+    arg(
+      "column (number, optional, default=0)",
+      _lt("The index of the column to be returned from within the reference range of cells.")
+    ),
+  ],
+  returns: ["ANY"],
+  compute: function (reference: MatrixArgValue, row: number = 0, column: number = 0): any {
```

Comment:
reference: ArgValue
row: PrimitiveArgValue = 0
column: PrimitiveArgValue = 0

return : FunctionReturnValue

Reply:
Need to apply the ```toNumber``` helper on ```row``` and ```column```. And use something like ```_row``` ```_column```  
in the rest of the function